etcd/src/main/scala/io/buoyant/etcd/Key.scala

```diff
@@ -0,0 +1,210 @@
+package io.buoyant.etcd
+
+import com.twitter.finagle.{Path, Service}
+import com.twitter.finagle.http._
+import com.twitter.io.Buf
+import com.twitter.util._
+import scala.collection.JavaConverters._
+
+class Key(key: Path, client: Service[Request, Response]) {
+
+  import Etcd._
+
+  def path: Path = key
+
+  def key(path: Path): Key = new Key(key ++ path, client)
+  def key(name: String): Key = key(Path.read(name))
+
+  private[this] val noParams = Seq.empty[(String, String)]
+
+  /** Get a key */
```

Comment:
Should aim for a consistent level of documentation on these operations.  Some have a lot, some have a little, some have none, some are scaladoc style, some aren't.